### ISSUE TITLE
FastAPI App **kwargs not recognized

### ISSUE THREAD

**Issue by JEub** [PythonNest/PyNest#28]
When attempting to initialize the App with a title, version, etc. the options fail to pass through to the parent class. 

Changing the example code in app.py to:

```python
app = App(
    description="PyNest service",
    modules=[
        ExamplesModule,
    ],
    title="PyNest Example",
)
```

The code returns an error:

`TypeError: App.__init__() got an unexpected keyword argument 'title'`

**Comment by JEub** [comment]
Correction ** This is resolved on the source code which has not been pushed to PyPI.

**Comment by JEub** [comment]
At what point will new versions of this package be pushed to PyPI?

**Comment by ItayTheDar** [comment]
Hey @JEub, a new version should be released in next week. I'll make sure that this issue will be fixed.

**Comment by ItayTheDar** [comment]
@JEub Thank you for raisin this issue! the fix is now available in version 0.1.1.
